ios-base/Makefile

```diff
@@ -1,8 +1,14 @@
+GIT_REPO = $(shell git rev-parse --show-toplevel)
+GIT_DIR = $(shell git rev-parse --git-dir)
+
 init:
 	mint run xcodegen
 	bundle install --path vendor/bundle
 	bundle exec pod install
 
+hooks:
+	@cp $(GIT_REPO)/ios-base/scripts/git-hooks/* $(GIT_DIR)/hooks
```

Comment:
I think this script should add to `init`. 🤔
If not add to `init`, please describe about this in `README.md` 🙏

Reply:
I'm wondering whether to put it in `init` because of a matter of taste, but deceide to put it💪